core/domain/state_domain.py

```diff
@@ -1319,6 +1342,55 @@ def __init__(
         # when the learner receives this outcome.
         self.missing_prerequisite_skill_id = missing_prerequisite_skill_id
 
+    def _register_all_translatable_fields(self) -> None:
+        """Registration of all translatable field for Outcome."""
+
+        if not self.feedback.html == '':
```

Comment:
Do we need this check here?

Reply:
Removed from here and added a check in ```translation_domain.BaseTranslatableObject``` class, before registering a translatble field.